examples/opensora_hpcai/opensora/pipelines/train_pipeline.py

```diff
@@ -247,12 +251,16 @@ def compute_loss(
         fps: Optional[Tensor] = None,
         ar: Optional[Tensor] = None,
     ):
-        t = ops.randint(0, self.diffusion.num_timesteps, (x.shape[0],))
-        noise = ops.randn_like(x)
+        if self.mode == 1:
+            t = ms.Tensor(np.random.randint(0, self.diffusion.num_timesteps, size=(x.shape[0],)), ms.int32)
+            noise = ms.Tensor(np.random.randn(*x.shape), ms.float32)
+        else:
+            t = ops.randint(0, self.diffusion.num_timesteps, (x.shape[0],))
```

Comment:
why can't use `ops` in the pynative mode?

Reply:
use np.random to speed up

Reply:
how about using `mint` then?